packages/fluentui/react-northstar/src/components/Dialog/Dialog.tsx

```diff
@@ -227,6 +229,38 @@ class Dialog extends AutoControlledComponent<WithAsProp<DialogProps>, DialogStat
     }
   };
 
+  handleBodyScroll(isFirstRender = false) {
+    if (isFirstRender && !this.props.defaultOpen) {
```

Comment:
```suggestion
    if (isFirstRender && !(this.props.defaultOpen || this.props.open)) {
```
@layershifter , do I understand correctly that dialog can be open by default by using either `defaultOpen` or `open={true}` interchangeably?

Reply:
`this.state.open` is SSOT for `AutoControlled` components, we covered this with @assuncaocharles already